Islandora-CLAW/carapace
# move to Github actions

# What does this Pull Request do?

replace travis with github actions
# What's new?
replace travis with github actions



# How should this be tested?

See if the build is successful
You can see the successful build over here: https://github.com/asulibraries/carapace/actions/runs/768103184

This includes the work from #24 

# Interested parties
@Islandora/8-x-committers
